yoda/llava_data_prep/scripts/edgar_ingestion.py

```diff
@@ -0,0 +1,35 @@
+import os
+import sys
+
+current_dir = os.path.dirname(os.path.abspath(__file__))
+kit_dir = os.path.abspath(os.path.join(current_dir, "../.."))
+repo_dir = os.path.abspath(os.path.join(kit_dir, ".."))
+
+sys.path.append(kit_dir)
+sys.path.append(repo_dir)
+
```

Comment:
Currently, there is no logging in this script. Please add logging for execution flow tracking as per your other scripts.